clients/src/main/java/org/apache/kafka/clients/admin/RemoveMembersFromConsumerGroupOptions.java

```diff
@@ -37,7 +38,15 @@ public RemoveMembersFromConsumerGroupOptions(Collection<MemberToRemove> members)
         this.members = new HashSet<>(members);
```

Comment:
As we have different semantics for an empty collection (it was "remove nothing" originally, and we change it to "remove all"), I am wondering if we should do a check if `members` is empty or not and throw an exception if empty? Or at least log a WARNING that empty implies "remove all" now?

Reply:
Make sense. It will throw exception if empty members provided now.